samples/client-sdk/getting-started-daily-js/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Daily Call Example</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div>
+      <label for="room-url">Room URL:</label>
+      <input
+        type="text"
+        id="room-url"
+        size="50"
+        placeholder="https://yourcompany.daily.co/hello"
+        value="https://mark.staging.daily.co/waiting_1"
```

Comment:
I think I'd suggest not filling it in with a valid default room URL, unless you _clearly_ call out that the user trying this demo may land in a room with a stranger in it.

Reply:
Thanks for catching this! 😅